.install-osx.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+brew update
+brew install fish nvm
+mkdir ~/.nvm
+curl -L https://github.com/oh-my-fish/oh-my-fish/raw/master/bin/install | fish
+fish -c 'omf install foreign-env'
+mkdir -p $HOME/.local/share/omf/pkg/nvm && cp -R * $HOME/.local/share/omf/pkg/nvm
+fish -c "nvm install v4.2.4; and nvm alias default v4.2.4"
```

Comment:
Ah, I see. I believe this is needed, as node isn't available on OS X out-of-box. My approach in this case is a little different: I always do a `brew install node` to have a default "system" version, then I install `nvm` and use it when I need distinct versions.